src/prefect/experimental/workers/base.py

```diff
@@ -25,6 +26,76 @@
 from prefect.utilities.dispatch import register_base_type
 
 
+class BaseJobConfiguration(BaseModel):
+    command: Optional[List[str]] = Field(template="{{ command }}")
+
+    @validator("command")
+    def validate_command(cls, v):
+        """Make sure that empty strings are treated as None"""
```

Comment:
This should be empty lists. Inside of the run functions, we are checking whether or not a `command` override is set. If it's not set, then we use the default command. We do this because you can't store `[sys.executable, "-m", "python"]` in json.

Reply:
I think I was worried about strings when it was going to be a `Union[str, list]`, and it's still somewhat unclear what we're going to use long-term.